server/controller/UserController.js

```diff
@@ -400,12 +401,13 @@ router.post(
       const user = await service.save(newUser);
       mailService.sendActivationMail(user);
 
-      res.json(user);
+      res.status(CREATED).json(user);
     } catch (error) {
       if (
         error.name?.includes("SequelizeUniqueConstraintError") &&
         error.original?.constraint === "Users_email_key"
       ) {
+        // TODO: Why no http constant at this point?
```

Comment:
Das habt "ihr" damals gemacht. Ich wasche meine Finger in Unschuld. Würde das aber hier auch angleichen.

Reply:
fixed